### ISSUE TITLE
Social Media Links to Share Information

### ISSUE THREAD

**Issue by echan-bridgephase** [BridgePhase/ctf#67]
Should we include commonly used social media platforms for visitors to share this information on social media? 

Links to share on FB, Twitter, LinkedIn, email, etc.

**Comment by jgarciarbridgephase** [comment]
umm.. I don't know, I defer to @belyea and @kgyovai

**Comment by kgyovai** [comment]
I think since this is a prototype we should focus more on the FDA API development aspects. If we had a full fledged version of the web app I would say adding social media links is a must. That's my 2 cents anyway. @belyea ?

**Comment by belyea** [comment]
yeah, I think its a great idea for a real website (like bridgephase.com!!!) but probably wouldn't score us any points with GSA so I say spend time elsewhere.
